README.md

```diff
@@ -52,6 +52,10 @@ Before attempting to get a copy of this, you should know(have) how to do the fol
 - Twitter: [ChrisboboSys](https://twitter.com/ChrisboboSys)
 - LinkedIn: [kiyong-christian](https://linkedin.com/in/kiyong-christian)
 
+👤 **Okoye Charles**
+
```

Comment:
- [ ][OPTIONAL] I think it would be better to add more profiles to have a great network you may add, LinkedIn and twitter